### PR TITLE
WB-1200.1: Add `wonder-blocks-birthday-picker` package

### PR DESCRIPTION
**Pull request by jandrade**
## Summary:

Added the wb-birthday-picker package

Issue: WB-1200

## Test plan:

N/A

(NOTE: I want to check if the check-size action fails this time)

### CODE REVIEW

**Review by jeresig — APPROVED** [review]

File: packages/wonder-blocks-birthday-picker/package.json
Comment (somewhatabstract) [inline packages/wonder-blocks-birthday-picker/package.json:3]: Since it'll get bumped on release, should we start new packages out at `0.0.0` instead?

File: packages/wonder-blocks-birthday-picker/package.json
Comment (somewhatabstract) [reply packages/wonder-blocks-birthday-picker/package.json:27]: Should it be exact like that, or should we allow variation in case webapp upgrades?

**Review by somewhatabstract — APPROVED** [review]
Some inline thoughts but nothing blocking.

Don't forget your changeset. :)

I'll have a poke at that action error. I swear I fixed it, but I must've missed something.

**Comment by somewhatabstract** [comment]
> Some inline thoughts but nothing blocking.
> 
> Don't forget your changeset. :)
> 
> I'll have a poke at that action error. I swear I fixed it, but I must've missed something.

@jandrade can you wait until this branch is reviewed and landed? That way we can see if it fixes the action error you see in this PR?

https://github.com/Khan/wonder-blocks/pull/1319?w=1

File: packages/wonder-blocks-birthday-picker/package.json
Comment (jandrade) [reply packages/wonder-blocks-birthday-picker/package.json:23]: I think it can be any either peer or regular deps. That's how we have handle it with the other existing WB components (at least for the internal wb packages).

File: packages/wonder-blocks-birthday-picker/package.json
Comment (jandrade) [reply packages/wonder-blocks-birthday-picker/package.json:27]: mmm good point, I'll include the ^ in that case.

**Pull request merged**
